fluxacademy.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<head>
+<title>Flux Academy</title>
+    <style>
+        body {
+            margin: 0;
+        }
+        .container {
+         padding: 1%;
+            box-sizing:border-box;
+            
+    }
+        .header {
+            height: 100px;
+            text-align:center;
+    }
+
+        .left-col, .right-col {
+        width: 25%;
+        float: left;
+    }
+        .center-col {
+            width:50%;
+            float:left;
+        }
+
+</style>
+</head>
+<body>
+    <img src="">
+    <!-- sticky nav how to -->
+    <nav>
+        <img src="" alt="logo">
+        <ul>
+            <li>Courses</li>
+            <li>Student Stories</li>
+            <li>About</li>
+            <li>Log In</li>
+        </ul>
+    </nav>
+    <div>
+        <h1>BECOME A FREELANCE WEB DESIGNER NO MATTER WHERE YOU LIVE</h1>
+        <p>You bring the creativity and passion. We teach you the skills you need to succeed as a freelance web designer. It all happens online, no matter where you are.</p>
+    </div>   
+    <div>
+        <img src="" alt="student"> 
+        <!--img overlaps video -->
+        <video width="" height="">
+            <source src="movie.mp4" type="video/mp4">
+        </video>
+    </div>
+    <div>
+        <h2>BUILD HIGH-VALUE SKILLS WITH BUSINESS AND WEB DESIGN COURSES ONLINE</h2>
+        <p>Thriving as a freelancer takes both craft and business strategy. Our courses help you with both. Become a skilled web designer and run a business you love with our proven processes and roadmaps</p>
+        <video width="" height="">
+            <source src="movie2.mp4" type="video/mp4">
+        </video>
+    </div>
+        <!--img overlaps video -->
+        <img src="" alt="student">
+    </div> 
+    <div>
+        <h3>The Flux Method</h3>
+        <p1>We teach from experience. Our educators are top-tier freelance web designers who will show you the frameworks they use in their own businesses.</p1>
+        <img src="" alt="cyclist">
+        <h3>LEARN AT YOUR OWN PACE</h3>
+        <p>Learn on your terms, but never alone. Enjoy lifetime access to your courses and the Flux Community.</p>
```

Comment:
Doesn't lines 65 to 67 need to be contained with in a div?

Reply:
How do we know that it should belong in it’s own div versus belonging to the div holding “The Flux Method.”

Reply:
Look at lines 78-81.
Those items are grouped within a `<div>`
For similar reasons, these three lines (65-67) need to be in a `<div>`
They are a group of HTML elements that belong together.